index.html

```diff
@@ -319,64 +338,155 @@ <h2>The <dfn>PerformanceObserver</dfn> interface</h2>
       <section>
         <h2><dfn>observe()</dfn> method</h2>
         <p>The <a>observe()</a> method instructs the user agent to
-        <dfn>register the observer</dfn> and must run these steps:</p>
+        <dfn>register the <var>observer</var></dfn> and must run these steps:</p>
         <ol data-link-for="PerformanceObserverInit">
-          <li>Let <var>entry types</var> be <var>options</var>
-          <a>entryTypes</a> sequence.
-          </li>
-          <li>Remove all unsupported types from <var>entry types</var>. The
-          user agent SHOULD notify developers if <var>entry types</var> is
-          modified. For example, a console warning listing removed types might
-          be appropriate.</li>
-          <li>If the resulting <var>entry types</var> sequence is an empty
-          sequence, abort these steps. The user agent SHOULD notify developers
-          when the steps are aborted to notify that registration has been
-          aborted. For example, a console warning might be appropriate.</li>
-          <li>If the <a>list of registered performance observer objects</a> of
-          <a data-cite="HTML/webappapis.html#concept-relevant-global">relevant
-          global object</a> contains a <a>registered performance observer</a>
-          whose `observer` is the <a data-cite=
-          "WHATWG-DOM#context-object">context object</a>, replace its
-          `options`, with <var>options</var>.
-          </li>
-          <li>Otherwise, append a new <a>registered performance observer</a>
-          object to the <a>list of registered performance observer objects</a>
-          of <a data-cite=
-          "HTML/webappapis.html#concept-relevant-global">relevant global
-          object</a>, with the <a data-cite="WHATWG-DOM#context-object">context
-          object</a> as `observer` and <var>options</var> as the `options`.
-          </li>
-          <li>If <var>options</var>' <a>buffered</a> flag is set, for each
-          <var>entry type</var> in <var>entry types</var> sequence:
+          <li>If <var>options</var>'s <a>entryTypes</a> and <a>type</a> members
+          are both omitted, then <a data-cite="!WEBIDL#dfn-throw">throw</a> a
+          <a data-cite="!WEBIDL#syntaxerror"><code>SyntaxError</code></a>.</li>
```

Comment:
Sanity check: is SyntaxError the best fit here?

Reply:
Hmm I'm using it because this is the type of error used in https://w3c.github.io/user-timing/#mark-method and it seems similar. Any thoughts on what type of error could be better?

Reply:
Fair enough. I don't have a competing alternative, just wanted to flag for other reviewers in case there is a better option here.